### PR TITLE
Fix NotYetAutoGenerated folder out of date issue by regenerate and sync the changes

### PR DESCRIPTION
**Pull request by shaopeng-gh**
The NotYetAutoGenerated folder was out of date, 
because I will be using the updated JSchema to regenerate, these out of sync issues will be also mixed in my PR.
It will be easier for reviewer to review by a separating PR:
  A PR which did not do any manual change, just make the current code up to date with current JSchema, which should have been done with the previous JSchema change.
  Another PR to use the next version of JSchema, for the BigInteger change.

This PR is purely just to make the NotYetAutoGenerated folder up to date with current JSchema, no other changes.

Did not include test because the changes are:
1. add `virtual` which was tested in all files those are not in NotYetAutoGenerated folder
2. comment updates

### CODE REVIEW

File: src/ReleaseHistory.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 * BUGFIX: Remove duplicated rule definitions when executing `match-results-forward` commands for results with sub-rule ids. [#2486](https://github.com/microsoft/sarif-sdk/pull/2486)
 * BUGFIX: Update `merge` command to properly produce runs by tool and version when passed the `--merge-runs` argument. [#2488](https://github.com/microsoft/sarif-sdk/pull/2488)
 * BUGFIX: Eliminate `IOException` and `DirectoryNotFoundException` exceptions thrown by `merge` command when splitting by rule (due to invalid file characters in rule ids).
+* BUGFIX: Fix NotYetAutoGenerated folder out of date issue by regenerate and manually sync the changes. [#2537](https://github.com/microsoft/sarif-sdk/pull/2537)
```
Comment (marmegh) [inline src/ReleaseHistory.md:9]: >Fix NotYetAutoGenerated folder out of date issue by regenerate and manually sync the changes. [](http://example.com/codeflow?start=10&length=93)

How does this bug present from the customer's perspective? BUGFIX release history details should be of the format:
`BUGFIX: Fix <symptom from customer's perspective> [by <change details if appropriate>]...`

Comment (shaopeng-gh) [reply src/ReleaseHistory.md:9]: right, the "NotYetAutoGenerated folder out of date" is not from customer's perspective.
how about:
BUGFIX: Fix classes inside NotYetAutoGenerated folder missing `virtual` keyword for public methods and properties, by regenerate and manually sync the changes.

File: src/ReleaseHistory.md [hunk 2]
```diff
@@ -5,7 +5,8 @@
 * BUGFIX: Resolve issue where `match-results-forward` command fails to generate VersionControlDetails data. [#2487](https://github.com/microsoft/sarif-sdk/pull/2487)
 * BUGFIX: Remove duplicated rule definitions when executing `match-results-forward` commands for results with sub-rule ids. [#2486](https://github.com/microsoft/sarif-sdk/pull/2486)
 * BUGFIX: Update `merge` command to properly produce runs by tool and version when passed the `--merge-runs` argument. [#2488](https://github.com/microsoft/sarif-sdk/pull/2488)
-* BUGFIX: Eliminate `IOException` and `DirectoryNotFoundException` exceptions thrown by `merge` command when splitting by rule (due to invalid file characters in rule ids).
+* BUGFIX: Eliminate `IOException` and `DirectoryNotFoundException` exceptions thrown by `merge` command when splitting by rule (due to invalid file characters in rule ids). [#2513](https://github.com/microsoft/sarif-sdk/pull/2513)
+* BUGFIX: Fix classes inside NotYetAutoGenerated folder missing `virtual` keyword for public methods and properties, by regenerate and manually sync the changes. [#2537](https://github.com/microsoft/sarif-sdk/pull/2537)
```
Comment (michaelcfanning) [inline src/ReleaseHistory.md:9]: Try this, which is a little more focused on customer impact:

Restore missing 'virtual' keyword for several SARIF types, previously overwritten by build system

**Review by michaelcfanning — APPROVED** [review]
:shipit:

**Pull request merged**